drivers/tests/UNITTESTS/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) 2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 add_subdirectory(doubles)
+add_subdirectory(AnalogIn)
```

Comment:
For discussion:
Do we want to only build the empty tests when we are building with the COVERAGE option set? This saves us building all these untested files for no reason on a routine unit test build.

If so:
We could make inclusion of all of the empty test directories conditional
```
if(COVERAGE)
    add_subdirectory(AnalogIn)
    add_subdirectory(SomeOtherComponent)
endif()
```
Or put the conditional test in the CMakeLists.txt for the individual test file, which has the benefit of confining to that test directory the changes that need to be made when actually implementing the unit test.
```
if(COVERAGE)
    include(GoogleTest)
    ...
    gtest_discover_tests(${TEST_NAME} PROPERTIES LABELS "drivers")
endif()
```

Reply:
I think adding this makes it slightly more annoying to add the real unit tests later, compromising our goal of having this be useful not just for coverage metrics, but also as a decent starting point for adding the tests.

Reply:
Yes I'm inclined to agree, and streamlining/encouraging the adding of tests is the primary goal.